src/config.js

```diff
@@ -76,7 +76,7 @@ const tagIds = {
     process.env.TASK_COMPLETED_TAG_ID || '1203085046769262592',
   addPointTagId: process.env.ADD_POINT_TAG_ID || '1258801833191669802',
   boostedPointTagId:
-    process.env.ADD_BOOSTED_POINT_TAG_ID || '1263873487953592381',
+    process.env.ADD_BOOSTED_POINT_TAG_ID || '&1263873487953592381',
```

Comment:
commonly we only save the numbers, maybe you can concat the `&` when using the variable in the code 
```suggestion
    process.env.ADD_BOOSTED_POINT_TAG_ID || '1263873487953592381',
```